### ISSUE TITLE
Explosions against VTOLS

### ISSUE THREAD

**Issue by Necrofox91** [MegaMek/megamek#1128]
Okay, my Peregrine is adjacent to the exploding enemy unit, but shouldn't the fact that I'm 5 hexes up protect me from the explosion??? I could understand a PSR check due to the shockwave, but if it affects the VTOL at that height then that would mean the explosion was 5-7 hexes, depending on how you count, large instead of just one.
Err maybe that's how it would work RAW, but it just doesn't make sense to me so... bug/enhancement report?

**Comment by BLOODWOLF333** [comment]
Yes ammo explosion damage should only hurt units within 1-hex side to side or above so, bug/this situation hasnt come up before and is an oversight.

**Comment by Necrofox91** [comment]
Okay, good. Didn't mess anything up then.

**Comment by Sleet01** [comment]
We've implemented rules-complete AE blast shaping in 0.50.04 so this should now be working correctly; closing.
